INSTALL.txt

```diff
@@ -11,7 +11,7 @@ REQUIREMENTS
 Oxwall Requires:
 
 - Apache WEB server, version 2.0 or higher. ( http://httpd.apache.org/download.cgi )
-- PHP 5.2.6 or higher. ( http://www.php.net/ )
+- PHP 5.5 or higher. ( http://www.php.net/ )
```

Comment:
we support 5.6 and higher. Please make this change and I will approve your request